src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -84,10 +84,10 @@ <h2 class="card-pf-title">
                           {{mission.name}}
                         </div>
                         <div class="list-group-item-text">
-                          <div *ngIf="mission.showMore !== true">
+                          <div *ngIf="mission.showMore !== true && mission.description">
```

Comment:
There probably should not be a 'more' link shown at all if there is no description.